components/create-post-page/BookPreviewCard.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  Box,
+  HStack,
+  VStack,
+  Wrap,
+  Tag,
+  TagLeftIcon,
+  Text,
+  Image,
+} from "@chakra-ui/react";
+import { AcademicCapIcon } from "@heroicons/react/outline";
+import {
+  resolveImageUrl,
+  resolveBookTitle,
+} from "@lib/helpers/frontend/resolve-book-data";
+import { PopulatedBook } from "@lib/services/book";
+
+interface BookPreviewProps {
+  book: PopulatedBook;
+  alternativeImage?: string;
+}
+
+export default function BookPreviewCard({
+  book,
+  alternativeImage,
+}: BookPreviewProps) {
+  return (
+    <Box
+      width="100%"
+      height="36"
+      shadow="md"
+      borderRadius="lg"
+      overflow="hidden"
+      background="secondaryBackground"
+    >
+      <HStack spacing="4" height="100%">
+        <Image
+          src={alternativeImage ?? resolveImageUrl(book)}
+          alt=""
+          height="100%"
+          mr="1"
+        />
+        <VStack
+          align="start"
+          height="100%"
+          py="4"
+          pr="4"
+          justifyContent="space-between"
+        >
+          <Box mb="2">
+            <Text fontWeight="semibold">{resolveBookTitle(book)}</Text>
```

Comment:
Content is being cut out if too long.

![Screen Shot 2022-03-29 at 10 17 02 PM](https://user-images.githubusercontent.com/40670885/160737490-8602cfd5-a7c0-4dca-bb76-e0c003750873.png)
![Screen Shot 2022-03-29 at 10 14 38 PM](https://user-images.githubusercontent.com/40670885/160737492-37b8551c-b228-41d8-99c8-34c4bc9457f7.png)